lib/src/uptech_growthbook_wrapper_test_client.dart

```diff
@@ -5,7 +5,7 @@ import 'package:growthbook_sdk_flutter/growthbook_sdk_flutter.dart';
 class UptechGrowthBookWrapperTestClient extends BaseClient {
   UptechGrowthBookWrapperTestClient({this.seeds});
 
-  final Map<String, bool>? seeds;
+  final Map<String, dynamic>? seeds;
```

Comment:
The `seeds` map needed to be updated in this way (and everywhere it's used) to support seeding features where the values are not boolean